services/project-grafana-loki/0.48.5/defaults/cm.yaml

```diff
@@ -6,6 +6,29 @@ metadata:
   namespace: ${releaseNamespace}
 data:
   values.yaml: |
+    ####################################################################
+    ## BEGIN DKP specific config overrides                            ##
+    ## This is added as a workaround to use the same configmap for    ##
+    ## both project-grafana-loki & object-bucket-claims helmreleases. ##
+    ####################################################################
+    dkp:
+      # these 2 OBC chart default values are irrelavent for project loki
```

Comment:
Hmmm it’s a little confusing now to have the velero and Loki blocks defined by default on the OBC chart default values, I wonder if we should just remove these as defaults from the chart values now that we need to use this chart in another location.

https://github.com/mesosphere/kommander-applications/pull/685/files#diff-952ccdf17c6e0a0e5f224751864ce2e58aaeced4a08a5d2c3808f0ec5b2bd255R140
looks like we are defining those blocks explicitly anyways so I would vote to not populate the OBC with any items by default

Reply:
makes sense. going to update this and rook-ceph-cluster